### ISSUE TITLE
Only allow API View links in API View section of scheduling tool

### ISSUE THREAD

**Issue by ronniegeraghty** [Azure/azure-sdk-tools#7592]
Change the text under the API View link to say API View links are required instead of preferred: 

![Image](https://github.com/Azure/azure-sdk-pr/assets/28957151/4b39b3e5-dc19-4776-aa4b-e7b19fdd444b)

Add link checking to only allow API View links to be added to this section.

**Comment by maririos** [comment]
@ronniegeraghty cruious. Are people uploading other content?

**Comment by ronniegeraghty** [comment]
> @ronniegeraghty cruious. Are people uploading other content?

Yup. I say the most common other link type is links to PRs for either the language repos or the specs repo. But I have also seen teams link to SharePoint folders there as well.
